sdk/src/v0.4/LaunchpadClient.ts

```diff
@@ -153,10 +154,11 @@ export class LaunchpadClient {
         tokenMint,
         tokenMetadata,
         tokenMetadataProgram: MPL_TOKEN_METADATA_PROGRAM_ID,
+        payer,
       })
       .preInstructions([
         createAssociatedTokenAccountIdempotentInstruction(
-          launchAuthority,
+          payer,
```

Comment:
This is changed because it should be the payer (gas sponsor), not the actual launch authority itself.
Rationale: Launch authority should simply be the one that signs off the launch (init, start), e.g. it could be a non-funded keypair.